package.json

```diff
@@ -102,8 +102,32 @@
       },
       {
         "command": "tomcat.server.browse",
-        "title": "Open in Browser",
+        "title": "browse",
         "category": "Tomcat"
+      },
+      {
+        "command": "tomcat.server.delete.context",
```

Comment:
What's the reason of suffixing the following command names with '.context'?

Reply:
In command palatte, "Start Tomcat Server" is a good command name, but for context commands, 'start' is kind of better when users right-click the Tomcat Server.

VSCode does not support different titles for same command, so .context commands are duplicate purposely for better naming context commands.